Makefile

```diff
@@ -242,7 +242,7 @@ docker-build-all: docker-build-debug docker-build-hermes
 ###                                Linting                                  ###
 ###############################################################################
 golangci_lint_cmd=golangci-lint
-golangci_version=v1.52.2
+golangci_version=v1.53.2
```

Comment:
@faddat we had a bump to golangci-lint here https://github.com/cosmos/gaia/pull/2562
I closed that because upgrading to that version caused the following error:

❯ make lint
--> Running linter
ante/ante.go:29:10: opts.AccountKeeper undefined (type HandlerOptions has no field or method AccountKeeper) (typecheck)
	if opts.AccountKeeper == nil {
	        ^
ante/ante.go:32:10: opts.BankKeeper undefined (type HandlerOptions has no field or method BankKeeper) (typecheck)
	if opts.BankKeeper == nil {
If I update the go.mod file, with the version version you specify, that change results in the above error.
Whats the different between the make lint target and changing the go.mod file?

Reply:
it's odd.  I don't see golangci-lint in the go.mod file anymore?